### PR TITLE
Add (tx *Tx) IsCommitted() method

### PR DESCRIPTION
**Pull request by TatsuNet**
Add method to get transaction status.

### CODE REVIEW

**Comment by goccy** [comment]
# [Codecov](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=h1) Report
> Merging [#6](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=desc) into [master](https://codecov.io/gh/knocknote/rapidash/commit/509263d875d8508039eacf9f898688049b4e364b?src=pr&el=desc) will **increase** coverage by `<.01%`.
> The diff coverage is `100%`.

[Impacted file tree graph](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master       #6      +/-   ##
==========================================
+ Coverage   71.13%   71.13%   +<.01%     
==========================================
  Files          21       21              
  Lines        8501     8503       +2     
==========================================
+ Hits         6047     6049       +2     
  Misses       1750     1750              
  Partials      704      704
```


| [Impacted Files](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [rapidash.go](https://codecov.io/gh/knocknote/rapidash/pull/6/diff?src=pr&el=tree#diff-cmFwaWRhc2guZ28=) | `71.8% <100%> (+0.1%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=footer). Last update [509263d...6c877fd](https://codecov.io/gh/knocknote/rapidash/pull/6?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: tx_test.go [hunk 1]
```diff
@@ -334,8 +334,14 @@ func TestTx_CreateByTableContext(t *testing.T) {
 		tx, err := cache.Begin(txConn)
 		NoError(t, err)
 		defer func() { NoError(t, tx.RollbackUnlessCommitted()) }()
+		if tx.IsCommitted() {
+			t.Fatal("unexpected transaction status")
+		}
 		NoError(t, tx.Commit())
 
+		if !tx.IsCommitted() {
+			t.Fatal("unexpected transaction status")
```
Comment (kanataxa) [inline tx_test.go]: I suggest to use `Equal()` or `Equalf()`. 
What do you think it?

Comment (TatsuNet) [reply tx_test.go]: Thanks!
https://github.com/knocknote/rapidash/pull/6/commits/61d7b3a6cbee8ad559cab9e347a9b77d371ea51b

**Comment by kanataxa** [comment]
LGTM !

**Comment by goccy** [comment]
LGTM f

**Pull request merged**